bot/utility/utility.py

```diff
@@ -95,6 +96,34 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.command(name='howto')
+    @command_log
+    async def howto(self, ctx: commands.Context, what: Optional[str] = None):
+        """Handler for the `howto code` command.
```

Comment:
Just for the howto command. "Code" is just currently the only subcommand, but the handler stays the same, even when we add new ones, so we don't want to need to change the docstring then.